test/test_land_cover.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from onsset import SettlementProcessor
+
+from pandas import DataFrame,Series
+from pandas.testing import assert_frame_equal, assert_series_equal
+from pytest import fixture
+
+
+class TestSettlementProcessor:
```

Comment:
You could move all the tests into one file, and one test class. Just add the methods to the `TestSettlementProcessor` class.  If you do this, you'll need to remove the duplicate `setup_dataframe` fixture methods. You could then just replace line 30 in this case with the lines 22-25.